hedera-node/hedera-app/src/main/java/com/hedera/node/app/hints/impl/HintsLibraryImpl.java

```diff
@@ -22,7 +22,7 @@ public class HintsLibraryImpl implements HintsLibrary {
 
     @Override
     public Bytes newCrs(final int n) {
-        return Bytes.wrap(BRIDGE.initCRS(n));
+        return Bytes.wrap(BRIDGE.initCRS((short) n));
```

Comment:
Please advice is this is the right thing to do.

Reply:
We could change the signature of `newCrs` to use short instead.

```
public Bytes newCrs(final short n) {

Reply:
I did the suggested signature change.